wormchain/x/wormhole/keeper/msg_server_register_account_as_guardian.go

```diff
@@ -25,6 +25,7 @@ func (k msgServer) RegisterAccountAsGuardian(goCtx context.Context, msg *types.M
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
Can you update the comments above the function to be accurate? With this change, the "special case" is no longer true.

Reply:
Yes, I'll update those comments now